app/src/common/shared/com/igalia/wolvic/utils/EnvironmentsManager.java

```diff
@@ -150,6 +150,7 @@ private void downloadEnvironment(@NonNull String envId) {
             // option in the radiobuttons widget (this cause the download item to be removed) or by the 'setOrDownloadEnvironment'
             // method during the java initialization invoked by the VRBrowser native code.
             Log.w(LOGTAG, "Download is still in progress; we shouldn't reach this code.");
+            assert false;
```

Comment:
Hmm not sure about this one. Couldn't it be triggered by a user with a slow connection that quickly switches between two environments several times?

I mean, if it's a valid (although unlikely) code path we should not have the assert. The assert should protect us against our own coding errors, not against user actions.

Reply:
Yes, you are right. I guess it will be the same scenario for https://github.com/Igalia/wolvic/pull/1131#discussion_r1450748131

Will remove both of the assertions.